pages.pl/windows/set.md

```diff
@@ -0,0 +1,20 @@
+# set
+
+> Wyświetl lub ustaw zmienne środowiskowe dla bieżącej instancji CMD.
+> Więcej informacji: <https://learn.microsoft.com/windows-server/administration/windows-commands/set>.
+
+- Lista wszystkich bieżących zmiennych środowiskowych:
+
+`set`
+
+- Ustaw zmienną środowiskową na określoną wartość:
+
+`set {{name}}={{value}}`
```

Comment:
Can you translate `{{name}}` and `{{value}}`?

Reply:
The values have been updated